README.md

```diff
@@ -4,4 +4,4 @@ Este repositorio ha sido creado con el fin de alojar el proyecto de la asignatur
 
 ## Descripción del proyecto
 
-En este proyecto se desarrollará un software que permitirá al usuario alquilar cualquier tipo de vehiculo disponible, ya sea con un conductor especializado en el caso de que el usuario no cuente con el permiso correspondiente para el tipo de vehiculo o bien sin un conductor para que sea el propio usuario el que se ponga a los mandos.
\ No newline at end of file
+En este proyecto se desarrollará una plataforma "Deportiva" donde los usuarios serán capaces de registrarse (pagando una cuota mensual), con el objetivo de obtener consejos de un coach especializado en el deporte que busque practicar dicho usuario. Además la propia plataforma podrá calcular una dieta óptima para el usuario, en función del objetivo que quiera lograr (por ejemplo, ganar músculo o perder peso), esta dieta estará sujeta a supervisión de especialistas en nutrición. Por último también se ofertará una tienda online donde podrán comprar suplementos alimenticios (proteinas, sales minerales, etc...). 
```

Comment:
¿Cuál es la propuesta de negocio? ¿Los "consejos" o la "dieta"? ¿De qué forma los dos se benefician de datos comunes? ¿En base a qué datos va a calcular la dieta?¿Qué tiene que ver la supervisión con la lógica de negocio y esta asignatura? "Por último"... ¿qué aporta a la lógica de negocio o a esta asignatura?
Por favor, tratad de entender qué se os pide, que es simplemente que planteéis un problema y los resolváis. Si lo que planteáis no resuelve ningún problema, ¡no lo incluyáis! Por eso tenéis que concentraros en una simple lógica de negocio, una sola, que resuelva un problema, uno solo (y que lo haga en la nube). Tienda + dieta + "consejos" no son la solución a un solo problema.

Reply:
Por como lo veo, la parte de consejos + dieta serían dos soluciones complementarias al mismo problema Si dejara solo la parte de las dietas, que es la que tiene el cálculo ¿Sería una propuesta válida? ¿Siendo el problema la incertidumbre de los deportistas novatos o simplemente que el usuario quiera una dieta de cierta manera?

Reply:
Como te he dicho arriba:

> El objetivo de todo este proceso es que enviéis lo que consideréis conveniente y yo lo evalúe. Si consideras que es una idea válida, hazlo siguiendo las instrucciones que te he dado.